examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -569,9 +569,7 @@ def run_benchmarks_from_spec(spec: Dict[str, BenchmarkDict], args: argparse.Name
     if args.custom_metrics_files is not None:
         import_metrics_hooks_files(args.custom_metrics_files)
     with open(output_log_path, "w", buffering=1) as listener:
-        print("#" * 80)
```

Comment:
You dont like the #########? No problem, just curious!

Reply:
No, they could be ok but they do not work.
They just appeared out of sync with the trace in between (check logs).
I.e. print() and logger.info() are out of sync on the screen by multiple lines.
So I think that these #### traces do not seem to be useful.